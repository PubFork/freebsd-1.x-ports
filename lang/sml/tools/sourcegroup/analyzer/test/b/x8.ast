- Parse.parseSource "b/x8.sml";
[closing b/x8.sml]
val it =
  SeqDec
    [MarkDec
       (StrDec
          [MarkStrb
             (Strb
                {constraint=NONE,
                 def=MarkStr
                       (StructStr
                          (SeqDec
                             [MarkDec
                                (ValDec
                                   [MarkVb
                                      (Vb
                                         {exp=MarkExp (IntExp 5,63,64),
                                          pat=VarPat [prim?]},61,64)],57,64),
                              MarkDec
                                (ValDec
                                   [MarkVb
                                      (Vb
                                         {exp=MarkExp
                                                (VarExp [prim?,prim?],73,76),
                                          pat=VarPat [prim?]},69,76)],65,76)]),
                        50,80),name=prim?},46,80)],36,80),
     MarkDec
       (StrDec
          [MarkStrb
             (Strb
                {constraint=NONE,
                 def=MarkStr
                       (StructStr
                          (SeqDec
                             [MarkDec
                                (ValDec
                                   [MarkVb
                                      (Vb
                                         {exp=MarkExp (IntExp 4,29,30),
                                          pat=VarPat [prim?]},27,30)],23,30)]),
                        16,34),name=prim?},12,34)],2,34)] : System.Compile.ast
- 