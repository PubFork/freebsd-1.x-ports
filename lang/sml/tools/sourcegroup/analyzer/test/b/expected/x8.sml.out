source sml "b/x8.sml"
  export structure A B ;

